utils/__tests__/validate_pr_quickstarts.test.js

```diff
@@ -0,0 +1,240 @@
+'use strict';
+
+const path = require('path');
+const { expect } = require('@jest/globals');
+
+const {
+  getQuickstartFromFilename,
+  getQuickstartConfigPaths,
+  buildMutationVariables,
+  buildUniqueQuickstartSet,
+  getGraphqlRequests,
+} = require('../validate_pr_quickstarts');
+const { readQuickstartFile } = require('../helpers');
+
+const mockDashboardRawConfiguration =
```

Comment:
would we be able to move this into a separate `.json` file that gets stringified?

```js
const mockDashboardRawConfigJSON = require('./mock_dashboard_config.json')
const mockDashboardRawConfig = JSON.stringify(mockDashboardRawConfigJSON)
```

Reply:
Yup, good idea. Much easier to read! I like the `mock_files` directory you introduced on the install plans PR. I've added that here: 00a23d79